sycl/test-e2e/bindless_images/examples/example_5_sample_cubemap.cpp

```diff
@@ -1,5 +1,4 @@
-// REQUIRES: aspect-ext_oneapi_cubemap
-// REQUIRES: build-and-run-mode
+// REQUIRES: target-nvidia, aspect-ext_oneapi_cubemap
```

Comment:
Similarly as above, the `target-nvidia` requirement was removed in https://github.com/intel/llvm/pull/16439. The aspect requirement should correctly filter out backends where this test is unsupported.

```suggestion
// REQUIRES: aspect-ext_oneapi_cubemap
```